css/style.css

```diff
@@ -31,11 +31,11 @@
     .tile > #projects > table p { margin:2px 0px 2px 0px; }
     .tile > .people {
       margin-top:75px;
-      text-align:justify; -ms-text-justify:distribute-all-lines; text-justify:distribute-all-lines;
+      text-align:left; -ms-text-justify:distribute-all-lines; text-justify:distribute-all-lines;
```

Comment:
Does text-align left conflict with the other alignment styles? Don't think -ms-text-justify and text-justify are still needed with align left.